docs/monitoring-aws-lambda.asciidoc

```diff
@@ -0,0 +1,69 @@
+[[monitoring-aws-lambda]]
+= Monitoring AWS Lambda Functions
+
+Elastic APM lets you monitor your AWS Lambda functions. 
+The natural integration of {apm-guide-ref}/apm-distributed-tracing.html[distributed tracing] into your AWS Lambda functions provides insights into the functions' execution and runtime behavior as well as their relationships and dependencies to other services.
+
+To get started with the setup of Elastic APM for your Lambda functions checkout the language-specific guides:
+
+* {apm-node-ref}/lambda.html[Quick Start with APM on AWS Lambda - Node.js]
+* {apm-py-ref}/lambda-support.html[Quick Start with APM on AWS Lambda - Python]
+* {apm-java-ref}/aws-lambda.html[Quick Start with APM on AWS Lambda - Java]
+
+Learn more about the <<aws-lambda-arch, architecture>> of Elastic APM for AWS Lambda. 
+
+[[aws-lambda-arch]]
+== APM Architecture for AWS Lambda
+
+AWS Lambda uses a special execution model to provide a scalable, on-demand compute service for code execution. In particular, AWS freezes the execution environment of a lambda function when there are no active requests being processed. This execution model poses additional requirements on APM in the context of AWS Lambda functions:
+
+1. To avoid data loss, APM data collected by APM agents needs to be flushed before the execution environment of a lambda function gets frozen.
```

Comment:
```suggestion
1. To avoid data loss, APM data collected by APM agents needs to be flushed before the execution environment of a lambda function is frozen.
```